### ISSUE TITLE
Error writing latest results

### ISSUE THREAD

**Issue by drewbitt** [promptfoo/promptfoo#96]
Error:
```
Failed to write latest results to C:\Users\andys-pc\.promptfoo\output\eval-2023-08-02T21-53-30.582Z.json:
Error: ENOENT: no such file or directory, lstat 'C:\Users\andys-pc\.promptfoo\output\latest.json'
```

I haven't created this directory

Could be something wrong near the `fs.symlinkSync`

**Comment by lox** [comment]
I rolled back to 0.17.9 with `npm install -g promptfoo@0.17.9` and it fixed the issue.

**Comment by typpo** [comment]
Thanks for the reports!  This _should_ be fixed in 0.18.1
